packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -436,6 +436,20 @@ class AndroidDevice extends Device {
     runCheckedSync(cmd);
     return true;
   }
+
+  @override
+  bool get supportsScreenshot => true;
+
+  @override
+  Future<bool> takeScreenshot(File outputFile) {
+    const String remotePath = '/data/local/tmp/screen.png';
```

Comment:
Might want to make this flutter-specific or random, but this is also OK.  does /data/local/tmp always exist?